src/components/gds-menu-item/gds-menu-item.scss

```diff
@@ -19,52 +14,36 @@
   @include desktopOnly {
     padding: 0 var(--spacing-s);
   }
-}
-
-// Vertical direction explicitly defined.
-:host-context(.vertical) {
-  & .content {
-    padding: 0 var(--spacing-m);
-  }
-}
-
-// Horizontal direction explicitly defined.
-:host-context(.horizontal) {
-  & .content {
-    padding: 0 var(--spacing-s);
-  }
-}
 
-.item {
   // For historical reasons, fallback to gds-label size="l"
   font-family: var(--gds-menu-item-font-family, var(--label-font-family));
   font-size: var(--gds-menu-item-font-size, var(--label-l-font-size));
   line-height: var(--gds-menu-item-line-height, var(--label-l-line-height));
   letter-spacing: var(--gds-menu-item-letter-spacing, var(--label-l-letter-spacing));
   font-weight: var(--gds-menu-item-font-weight, var(--label-l-font-weight));
   text-transform: var(--gds-menu-item-text-transform, var(--label-text-transform));
+  // Provides a possibility to offset text content when the font has incorrect
+  // vertical alignment for glyphs.
+  transform: var(--gds-menu-item-transform);
 }
 
-.underline-container {
-  position: relative;
-  width: 100%;
-  height: 0;
-  display: var(--menu-item-underline-display, block);
+// Active style
+:host([active]) {
+  text-decoration: var(--gds-menu-item-underline-active, underline solid currentColor 3px);
+  text-underline-offset: var(--gds-menu-item-underline-offset-active, 5px);
```

Comment:
Nice! Not sure why I did it with html before. 😄

Reply:
I wasn't even aware you could set the underline thickness until I looked it up.

Browser support isnt amazing but I think it's good enough for this case: https://caniuse.com/?search=text-decoration-thickness